--- a/package/feeds/kiddin9/base-files/files/etc/init.d/led
+++ b/package/feeds/kiddin9/base-files/files/etc/init.d/led
@@ -76,6 +76,8 @@ load_led() {
 		"netdev")
 			[ -n "$dev" ] && {
 				echo $dev > /sys/class/leds/${sysfs}/device_name
+				[ -e "/sys/class/leds/${sysfs}/mode" ]  && \
+					echo $mode > /sys/class/leds/${sysfs}/mode
 				for m in $mode; do
 					[ -e "/sys/class/leds/${sysfs}/$m" ] && \
 						echo 1 > /sys/class/leds/${sysfs}/$m
